Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj

```diff
@@ -44,21 +41,18 @@
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.7.0" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.1" />
   </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\..\..\Application\EdFi.Ods.Common\EdFi.Ods.Common.csproj" />
+    <None Include="../../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
```

Comment:
this should move the group with the other nones or contents

Reply:
Fixed it . Thanks